zip-0252.rst

```diff
@@ -110,24 +105,17 @@ Once NU5 activates on Testnet or Mainnet, NU5 nodes SHOULD take steps to:
 - reject new connections from pre-NU5 nodes on that network;
 - disconnect any existing connections to pre-NU5 nodes on that network.
 
-The change to the peer-to-peer protocol described in ZIP 239 takes effect
+The change to the peer-to-peer protocol described in ZIP 239 took effect
 from peer protocol version 170014 onward, on both Testnet and Mainnet. [#zip-0239]_
 
-The change to the peer-to-peer protocol described in ZIP 155 is currently
-proposed to take effect from peer protocol version 170017 onward, on both
-Testnet and Mainnet [#zip-0155]_. This might be changed to version 170016
-depending on scheduling constraints.
```

Comment:
ZIP 155 was never technically part of NU5, and progress on it has been somewhat slow (because we've been all-hands-on-deck for NU5 and haven't been able to help Zancas with it), so it will likely be its own independent protocol version bump.

Reply:
Zebra implemented compatibility parsing for the new messages in ZIP-155, and it's active right now.

But we don't send those messages.